### PR TITLE
Minor improvements in codes and formatting.

### PR DESCRIPTION
**Pull request by Yousha**

### CODE REVIEW

File: gamemodes/PPC_Trucking.pwn [hunk 1]
```diff
@@ -3,7 +3,11 @@
 
 #pragma unused ret_memcpy
 
-
+#if __Pawn >= 0x0400
+   #error This script does not support PAWN 4.0
```
Comment (CodeAlDente) [inline gamemodes/PPC_Trucking.pwn:9]: * and greater

**Review by CodeAlDente — APPROVED** [review]

File: gamemodes/PPC_Trucking.pwn [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 #pragma unused ret_memcpy
 
+#pragma semicolon 1
```
Comment (CodeAlDente) [inline gamemodes/PPC_Trucking.pwn:6]: I see your point but is this really necessary? I did not try it out yet but I guess PAWN is always yelling about missing semicolons even without that line.

File: gamemodes/PPC_Trucking.pwn [hunk 3]
```diff
@@ -139,6 +139,8 @@ public OnPlayerConnect(playerid)
 	if (IsPlayerNPC(playerid))
 		return 1;
 
+   SendDeathMessage(INVALID_PLAYER_ID, playerid, 200); // 200 = DEATHMESSAGE_CONNECT
```
Comment (CodeAlDente) [inline gamemodes/PPC_Trucking.pwn:142]: Please undo this change as we already have chat messages that inform about players who join / leave the server. If you want file an issue regarding the usage of `SendDeathMessage` in general.

File: pawno/include/PPC_Defines.inc [hunk 4]
```diff
@@ -240,6 +240,9 @@
 #define ADMIN_SPEC_TYPE_VEHICLE 2
 
 
+// Define strings
+#define MAX_STRING 128
```
Comment (CodeAlDente) [inline pawno/include/PPC_Defines.inc:244]: What do you mean by `MAX_STRING`? According to https://wiki.sa-mp.com/wiki/Limits there are two different character lenghts (Input: Chat/Commands / Output: e.g. `SendClientMessage`). I guess we should add both.

**Review by CodeAlDente — CHANGES_REQUESTED** [review]
Please see comments

File: pawno/include/PPC_PlayerCommands.inc [hunk 5]
```diff
@@ -1,27 +1,28 @@
 // This is just a joke command
-COMMAND:me(playerid,params[])
+COMMAND:me(playerid, params[])
```
Comment (CodeAlDente) [inline pawno/include/PPC_PlayerCommands.inc:2]: Please undo all changes to this lines as they are mainly a matter of personal preferences, e.g. leaving blank spaces or removing the quotes. Apart from that changes to the color of messages are always welcome. We need to make more use of the color definitions. That would require an own pull request.

File: gamemodes/PPC_Trucking.pwn [hunk 6]
```diff
@@ -409,6 +409,7 @@ public OnPlayerDisconnect(playerid, reason)
 		APlayerData[playerid][RentedVehicleID] = 0;
 	}
 
+   SendDeathMessage(INVALID_PLAYER_ID, playerid, 201); // 201 = DEATHMESSAGE_DISCONNECT
```
Comment (CodeAlDente) [inline gamemodes/PPC_Trucking.pwn:412]: Please undo this change as we already have chat messages that inform about players who join / leave the server. If you want file an issue regarding the usage of `SendDeathMessage` in general.

File: gamemodes/PPC_Trucking.pwn (hunk 2 above)
Comment (Yousha) [reply gamemodes/PPC_Trucking.pwn:6]: Explicit programming instead of leaving stuff in default... ?

File: pawno/include/PPC_PlayerCommands.inc (hunk 5 above)
Comment (Yousha) [reply pawno/include/PPC_PlayerCommands.inc:2]: Yea, I was thinking about this but i found different things:

`COMMAND:pm(playerid, params[])`

`COMMAND:createhouse(playerid, params[])`

`COMMAND:delhouse(playerid, params[])`
and...

As you can see there's one SPACE between them 
Only "me" and "givelicense" have no SPACE

So it is more like you forgot to change them(2)

File: gamemodes/PPC_Trucking.pwn (hunk 1 above)
Comment (Yousha) [reply gamemodes/PPC_Trucking.pwn:9]: What do you mean? `#if __Pawn >= 0x0400` there is already >= for v4 and greater

Comment (CodeAlDente) [reply gamemodes/PPC_Trucking.pwn:9]: Yes but the error message does not say that. It does only say that version 4 is not supported even when you are using a version higher than that.

File: gamemodes/PPC_Trucking.pwn (hunk 2 above)
Comment (CodeAlDente) [reply gamemodes/PPC_Trucking.pwn:6]: Fair point! Since it does not require extra configuration settings we can add it here. But please add a short description of it above in a comment. The title of the commit should work.

File: gamemodes/PPC_Trucking.pwn (hunk 3 above)
Comment (CodeAlDente) [reply gamemodes/PPC_Trucking.pwn:142]: Opened issue for this, see #87

File: gamemodes/PPC_Trucking.pwn (hunk 6 above)
Comment (CodeAlDente) [reply gamemodes/PPC_Trucking.pwn:412]: Opened issue for this, see #87

File: pawno/include/PPC_PlayerCommands.inc (hunk 5 above)
Comment (CodeAlDente) [reply pawno/include/PPC_PlayerCommands.inc:2]: Oh I see. Could you please add the blank space for `/givelicense` as well?

File: pawno/include/PPC_Defines.inc (hunk 4 above)
Comment (CodeAlDente) [reply pawno/include/PPC_Defines.inc:244]: Can you please change this to the following:

```pawn
#define MAX_STRING_INPUT_TEXT 128
#define MAX_STRING_OUTPUT_TEXT 144
```

Feel free to change it to a more suitable name if you would like to. Thanks!

File: gamemodes/PPC_Trucking.pwn (hunk 1 above)
Comment (Yousha) [reply gamemodes/PPC_Trucking.pwn:9]: Oh got it now. TY
